consensus/spos/bls/subroundSignature.go

```diff
@@ -158,6 +158,11 @@ func (sr *subroundSignature) receivedSignature(cnsDta *consensus.Message) bool {
 	}
 
 	currentMultiSigner := sr.MultiSigner()
+	err = currentMultiSigner.VerifySignatureShare(uint16(index), cnsDta.SignatureShare, sr.GetData(), nil)
+	if err != nil {
+		return false
```

Comment:
You can log the error,  just to know which is the error when this method returns false in a critical situation and to keep the pattern used in this method:

		log.Debug("receivedSignature.VerifySignatureShare",
			"index", index,
			"error", err.Error())

Reply:
done